helix-core/src/main/java/org/apache/helix/controller/stages/StateTransitionThrottleController.java

```diff
@@ -155,16 +136,20 @@ protected boolean shouldThrottleForResource(
    */
   protected boolean shouldThrottleForInstance(
       StateTransitionThrottleConfig.RebalanceType rebalanceType, String instanceName) {
+    return shouldThrottleForGivenMap(rebalanceType, instanceName, _pendingTransitionAllowedPerInstance);
```

Comment:
Is there a reason why you split this function into 2? Is there any consumer of shouldThrottleForGivenMap() ???